service_actor/app/repositories/dao/ActorDao.scala

```diff
@@ -47,10 +47,10 @@ class ActorDao @Inject() (
     db.run(query.result.headOption)
   }
 
-  def getByName(searchString: String): Future[Seq[Person]] = {
+  def getByName(mid: MuseumId, searchString: String): Future[Seq[Person]] = {
     val likeArg = searchString.toLowerCase
     val query = actorTable.filter { a =>
-      a.fn.toLowerCase like s"%$likeArg%"
+      (a.fn.toLowerCase like s"%$likeArg%") && a.museumId === mid.underlying
```

Comment:
Mmm...I also see we have a job to do to update the return types of the public methods here. But that's an issue in itself, really.